release.go

```diff
@@ -82,6 +95,18 @@ func (r Release) Version() string {
 	return r.version
 }
 
+func aggregateReleaseTimestamp(bundles []Bundle) (string, error) {
+	var t time.Time
+
+	for _, b := range bundles {
+		if b.Time.After(t) {
+			t = b.Time
+		}
+	}
```

Comment:
The rule is to use the latest version bundle timestamp to define the timestamp of the aggregated release.